src/compiler/scala/tools/nsc/typechecker/Namers.scala

```diff
@@ -1461,12 +1461,16 @@ trait Namers extends MethodSynthesis {
         // Note that matching MethodType of NullaryMethodType must be nilary not nelary.
         def overriddenNilary(sym: Symbol) = sym.info.isInstanceOf[MethodType]
         if (overridden != NoSymbol && vparamSymss.isEmpty && overridden.alternatives.exists(overriddenNilary)) {
-          if (settings.warnNullaryOverride)
-            context.unit.toCheck += {() =>
-              def javaDetermined(sym: Symbol) = sym.isJavaDefined || isUniversalMember(sym)
-              if (!meth.overrides.exists(javaDetermined))
-                context.warning(ddef.pos, "nullary method assumes an empty parameter list from the overridden definition", WarningCategory.LintNullaryOverride)
+          def notExempt() = !meth.overrides.exists(sym => sym.isJavaDefined || isUniversalMember(sym))
+          val msg = "nullary method assumes an empty parameter list from the overridden definition"
```

Comment:
Slightly cryptic...

Reply:
"assume" as in "take upon oneself", but I hadn't considered the secondary connotation of "feign". https://www.wordnik.com/words/assume

Reply:
Changed.  Re-comment on the new version if you think it needs tweaking.